webapp/src/main/resources/public/js/sdk/entity/Branch.js

```diff
@@ -0,0 +1,51 @@
+import User from "./User"
+import Repository from "./Repository";
+import BranchRepository from "./BranchRepository";
+import BranchStatisticScreenshot from "./BranchStatisticScreenshot";
+
+export default class Branch {
+    constructor() {
```

Comment:
Oh! you created objects for the client. We stopped using them saying it wasn't that great and instead we should have something like flow.

Not hurting to have them but just if you do something else later you can avoid writing those